docs/getting-started-scalardl-ledger.md

```diff
@@ -0,0 +1,538 @@
+   scalardl-ledger-envoy-84dcc85b6d-c2nnp      1/1     Running     0          5m55s
+   scalardl-ledger-ledger-57dcb56f58-dgcjs     1/1     Running     0          5m55s
+   scalardl-ledger-ledger-57dcb56f58-fdnvm     1/1     Running     0          5m55s
+   scalardl-ledger-ledger-57dcb56f58-mtrfc     1/1     Running     0          5m55s
+   schema-loader-ledger-schema-loading-cscr4   0/1     Completed   0          8m16s
+   ```
+   If the Scalar DL Ledger pods are deployed properly, you can see the STATUS are `Running`.  
+
+1. Check the Scalar DL Ledger Services are deployed.
+   ```console
+   $ kubectl get svc
+   NAME                            TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)                           AGE
+   kubernetes                      ClusterIP   10.96.0.1        <none>        443/TCP                           21h
+   scalardl-ledger-envoy           NodePort    10.98.162.209    <none>        50051:31452/TCP,50052:31118/TCP   6m23s
+   scalardl-ledger-envoy-metrics   ClusterIP   10.105.122.178   <none>        9001/TCP                          6m23s
+   scalardl-ledger-headless        ClusterIP   None             <none>        50051/TCP,50053/TCP,50052/TCP     6m23s
+   scalardl-ledger-metrics         ClusterIP   10.110.55.239    <none>        8080/TCP                          6m23s
+   ```
+   If the Scalar DL Ledger Services are deployed properly, you can see a private IP address in the CLUSTER-IP column. (Note: `scalardl-ledger-headless` has no CLUSTER-IP.)  
+
+1. (Optional) If you set `LoadBalancer` to `envoy.service.type` in the `scalardl-ledger-custom-values.yaml`, you can access Scalar DL Ledger from 127.0.0.1.  
+   To expose the `scalardl-ledger-envoy` service as your local `127.0.0.1:50051` and `127.0.0.1:50052`, open another terminal, and run the `minikube tunnel` command.
+   ```console
+   $ minikube tunnel
+   ```
+   After running the `minikube tunnel` command, you can see the  EXTERNAL-IP of the `scalardl-ledger-envoy` as  `127.0.0.1`.
+   ```console
+   $ kubectl get svc scalardl-ledger-envoy
+   NAME                    TYPE           CLUSTER-IP      EXTERNAL-IP   PORT(S)                           AGE
+   scalardl-ledger-envoy   LoadBalancer   10.98.162.209   127.0.0.1     50051:31452/TCP,50052:31118/TCP   9m7s
+   ```
+
+## Step 8. Start Client container
+
+In this section, we will create a Client container to run a sample contract of Scalar DL.  
+We will use certificate files in the Client container. So, mount ~/scalardl-test/certs to the Client container.  
+
+1. Start the Client container on the Docker Network `minikube`.
+   ```console
+   $ docker run -d --name scalardl-client --hostname scalardl-client -v ~/scalardl-test/certs:/certs --network minikube --entrypoint sleep ubuntu:20.04 inf
+   ```
+
+1. Check the Client container is running.
+   ```console
+   $ docker ps -f name=scalardl-client
+   ```
+
+## Step 9. Run Scalar DL sample contract in the Client container
```

Comment:
I think `contracts` is correct, because there are two contracts `StateUpdater` and `StateReader` in this step.
I fixed in a2e1c6bf4e9c564d8a10efb03e7138801f97c695.